migrations/20200927205755_user.js

```diff
@@ -0,0 +1,23 @@
+/* eslint-disable func-names */
+const { tableUser } = require('../src/models/user.js')
+
+exports.up = function (knex) {
+  return knex.schema.createTable(tableUser, (table) => {
+    table.increments('BeneficiaryId').primary()
+    table.integer('RefereeId').unique()
+    table.text('Name')
+    table.text('email').notNullable().unique()
+    table.text('phone').notNullable().unique()
+    table.text('Address')
+    table.text('occupation')
+    table.enum('MaritalStatus', ['Female', 'Male'])
+    table.decimal('Income')
+    table.integer('CaseId').notNullable().unique()
```

Comment:
Hmm... I think it's ok to just have the case link back to the beneficiary and referee. More flexible if at some point a beneficiary has multiple cases